### DISCUSSION TITLE
Still using?

### DISCUSSION THREAD

**Discussion by RaidenMK** [jeremyckahn/chitchatter#113] [Q&A]
Are people still using https://chitchatter.im ? Im using it,but all the existing public rooms that is added is empty

**Comment by jeremyckahn** [comment]
I don't personally spend much time in the [community rooms](https://github.com/jeremyckahn/chitchatter/wiki/Chitchatter-Community-Rooms) (I'm not much of a chatter, believe it or not), but I hope that others might use them! I'm happy to add more rooms to the list if folks have any requests.

**Reply by RaidenMK** [reply]
I might have some ideas of public rooms. i try think of some
